Makefile

```diff
@@ -49,6 +49,7 @@ fmt:
 
 # note: when running e2e tests manually be sure to set the e2e filename:
 # 	'export CI_E2E_FILENAME=rel-nightly'
+#	'export CHANNEL=nightly'
```

Comment:
Maybe provide default a value with `CHANNEL ?= nightly` ?

Reply:
setting the default in docker-compose file